docs/Tutorial_DAMP.ipynb

```diff
@@ -0,0 +1,569 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #39.        next_expected_chunk_size =  next_pow2(m) * (2 ** (n_chunks - 1)) 
</pre><p>@NimaSarajpoor</p><p>Add comment to explain <code>n_chunks - 1</code>  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/920/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Reply:
<p>Done.</p>